account_payment_term_surcharge/__manifest__.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+#
+#    Copyright (C) 2015  ADHOC SA  (http://www.adhoc.com.ar)
+#    All Rights Reserved.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Term Surcharge',
+    'version': "13.0.1.0.0",
+    'category': 'Accounting',
+    'sequence': 14,
+    'summary': 'Calculate and generate term surcharge for payments',
```

Comment:
Al name le pondría: Surcharges on payment terms
En summary: Allow to add surcharges for invoices on payment terms
(o algo así)